### PR TITLE
Push changes to Staging In Preparation for Final Submission

### PR DESCRIPTION
**Pull request by georgetayqy**
### Description

Let's move to push changes from the Dev branches to staging for actual testing in preparation of the final submission.

### Additional information

N/A

### CODE REVIEW

**Comment by georgetayqy** [comment]
CI is failing as there is an invalid file name with `:` in it.

File: .github/workflows/production.yml [hunk 1]
```diff
@@ -15,106 +18,328 @@ jobs:
...
+      - name: Install Frontend Node Dependencies
+        working-directory: peer-prep
+        run: npm ci
+
+      - name: Test Build on Frontend
+        working-directory: peer-prep
+        run: npm run build
+
+      - name: Test User Service
+        working-directory: user-sevice
```
Comment (georgetayqy) [inline .github/workflows/production.yml:41]: Change the typo here

File: .github/workflows/staging.yml [hunk 2]
```diff
@@ -22,108 +25,324 @@ jobs:
...
+      - name: Install Frontend Node Dependencies
+        working-directory: peer-prep
+        run: npm ci
+
+      - name: Test Build on Frontend
+        working-directory: peer-prep
+        run: npm run build
+
+      - name: Test User Service
+        working-directory: user-sevice
```
Comment (georgetayqy) [inline .github/workflows/staging.yml:43]: Change all instances of sevice to service

**Review by georgetayqy — COMMENTED** [review]
Typo is causing CI to fail, we need to fix them

**Review by pzl111 — APPROVED** [review]
LGTM LGTM

**Pull request merged**